src/style.css

```diff
@@ -1,3 +1,65 @@
+:root {
+  --active-color: #00acdc;
+}
+
+html {
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+}
+
 body {
+  background-color: #e5e5e5;
+  margin: 0;
+}
+
+.header {
+  background-color: white;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  position: relative;
+  padding: 0 50px;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
   margin: 0;
+  list-style: none;
+  padding-left: 0;
+}
+
+.nav__link {
+  display: flex;
+  align-items: center;
+  height: 60px;
+  margin-left: 20px;
+  color: black;
+  text-decoration: none;
+  font-weight: 500;
+  position: relative;
+}
+
+.is-active {
+  margin-left: 0;
+  color: var(--active-color);
+}
+
+.is-active::after {
+  content: '';
+  display: block;
+  width: 100%;
+  height: 4px;
+  background-color: var(--active-color);
+  position: absolute;
+  bottom: 0;
+  border-radius: 8px;
+}
+
+.nav__link:hover {
+  color: var(--active-color);
```

Comment:
The hover effect on `.nav__link` changes the text color to the active color. Ensure this is the desired behavior, as it might make the text less readable depending on the background color.